lib/type-helpers/intersection-type.helper.ts

```diff
@@ -7,6 +7,7 @@ import { DECORATORS } from '../constants';
 import { ApiProperty } from '../decorators';
 import { ModelPropertiesAccessor } from '../services/model-properties-accessor';
 import { clonePluginMetadataFactory } from './mapped-types.utils';
+import { inheritPropertyInitializers } from '@nestjs/mapped-types/dist/type-helpers.utils';
```

Comment:
This function should be exported from the root now

Reply:
In CircleCI Build is failed due:
```

lib/type-helpers/intersection-type.helper.ts:5:3 - error TS2305: Module '"../../node_modules/@nestjs/mapped-types"' has no exported member 'inheritPropertyInitializers'.

5   inheritPropertyInitializer
```

But I saw it is fixed by You in this commit https://github.com/nestjs/mapped-types/commit/95bf29188355f0043d6faf11106fab063b8300f5.

So I will change mapped-types to next minor version.